tests/torchtune/rl/test_parameter_server.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import importlib
+
+SKIP_FILE = not (
+    importlib.util.find_spec("ray") is not None
+    and importlib.util.find_spec("transformers") is not None
+    and importlib.util.find_spec("vllm") is not None
+)
+
+
+if not SKIP_FILE:
```

Comment:
@joecummings @ebsmothers @pbontrager is doing something like this acceptable?

I can't seem to put the @pytest.skip on the `TestParamServer` class itself and have all the imports inside, because if I don't put the @ray.remote decorators on `DummyTrainer`/`DummyCollector`, then I have trouble instantiating them like `ray.remote(num_cpus=1, num_gpus=1)(DummyCollector).remote()` when running the tests with pytest

it will error like

```
values     = [ActorDiedError(RayTaskError('__init__', 'Traceback (most recent call last):\n  File "python/ray/_raylet.pyx", line 18...class)\nModuleNotFoundError: No module named \'test_parameter_server\'\n'), 'ray::DummyTrainer.__init__', None, None))]
``` 

:/

Reply:
The dummy decorator approach looks good to me

Reply:
Separately another possibility is to just update some of our pytest configs to default to ignore these paths. Then we explicitly include them in the async RL CI job. (This is just a hunch, I am not 100% sure that using e.g. pytest's `--ignore-paths` will completely skip over any imports etc on those paths)